keylime/cert_utils.py

```diff
@@ -146,3 +190,33 @@ def verify_ek_script(script: Optional[str], env: Optional[Dict[str, str]], cwd:
         logger.error("Error while trying to run external check script to validate EK: %s", err)
         return False
     return True
+
+
+def iak_idevid_cert_checks(
+    idevid_cert: bytes, iak_cert: bytes, tpm_cert_store: str
+) -> Tuple[str, Optional[tpm2_objects.pubkey_type], Optional[tpm2_objects.pubkey_type]]:
+    idevid_cert_509 = x509_der_cert(idevid_cert)
+    iak_cert_509 = x509_der_cert(iak_cert)
+    # NEEDS CRYPTO >= 38.0.0
```

Comment:
Some sort of test case would be helpful, like a test case in `test/test_cert_utils.py`

Reply:
@Isaac-Matthews Wouldn't it be better to simply skip the test? This way, the test won't fail if the support is backported to an older version and the parsing start to work.

https://docs.python.org/3/library/unittest.html#skipping-tests-and-expected-failures

Reply:
Thanks I was not aware of this option. [Changed here](https://github.com/keylime/keylime/pull/1477/commits/16b0be9542b7c04111aa2d86a8d9a5b2285c0563)